### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: '/sdcard/etheroll/.config/pyethapp/keystore/'

### ISSUE THREAD

**Issue by AndreMiras** [AndreMiras/EtherollApp#134]
https://sentry.io/organizations/andre-5t/issues/994298505/?project=1111738&referrer=github_plugin

```
FileNotFoundError: [Errno 2] No such file or directory: '/sdcard/etheroll/.config/pyethapp/keystore/'
  File "main.py", line 211, in main
  File "main.py", line 63, in run
  File "main.py", line 155, in pull_accounts_rolls
  File "github.andremiras.etheroll/.buildozer/android/app/ethereum_utils.py", line 19, in get_account_list

FileNotFoundError: [Errno 2] No such file or directory: '/sdcard/etheroll/.config/pyethapp/keystore/'
```

**Comment by AndreMiras** [comment]
Happens on some devices on account creation.
Possibly linked to #133

**Comment by AndreMiras** [comment]
Fixed with #133
